src/lib/tabs/tab-body.ts

```diff
@@ -175,7 +220,7 @@ export class MdTabBody implements OnInit, AfterViewChecked {
   }
 
   /** Whether the provided position state is considered center, regardless of origin. */
-  private _isCenterPosition(position: MdTabBodyPositionState|string): boolean {
+  _isCenterPosition(position: MdTabBodyPositionState|string): boolean {
```

Comment:
Will need to remove the underscore if it's not private.

Reply:
Ack.